packages/ui/src/components/va-modal/VaModal.vue

```diff
@@ -11,6 +11,7 @@
     <teleport :to="attachElement" :disabled="$props.disableAttachment">
       <modal-element
         name="va-modal"
+        ref="modal"
```

Comment:
Maybe
```suggestion
```
and below in 20th line: `<div class="va-modal" v-if="valueComputed" ref="modal">`